KrispyTech/airneis
# AIRNEIS-112 : Frontend HTTP API + Config bugfixes

# DONE 

- Created a request functions 
- Created a function per HTTP method
- Created a function to get,post,delete,or update category
- Made a test endpoint to be sure those functions works

# TEST

- Create a .env.local file into your front folder
- Copy the content of .env.local.exemple into it 
- MAKE SURE YOU DON'T HAVE A CATEGORY WITH THE ID 127 
- Make a get request to this endpoint `localhost:3000/api/test/categoriesFunctions`
